vllm_ascend/platform.py

```diff
@@ -147,39 +148,7 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         else:
             enforce_eager = getattr(model_config, "enforce_eager", False)
 
-        if additional_config is not None:
-            enable_graph_mode = additional_config.get("enable_graph_mode",
-                                                      False)
-            if enable_graph_mode:
-                if enforce_eager:
-                    raise RuntimeError(
-                        "Can't enable graph mode and eager mode at the same time. Please set `enforce_eager=False` if you attempt to enable NPU graph mode."
-                    )
-                elif envs.VLLM_USE_V1 and envs.VLLM_MLA_DISABLE:
-                    logger.warning(
-                        "NPU graph mode is still experimental and not supported for V1 without mla currently, "
-                        "it has been disabled automatically.")
-                    additional_config["enable_graph_mode"] = False
-                if model_config:
-                    model_type = model_config.hf_config.model_type
-                    if "deepseek" not in model_type:
-                        raise NotImplementedError(
-                            "enable_graph_mode only works with deepseek model."
-                        )
-
-        elif envs.VLLM_USE_V1 and model_config is not None and not enforce_eager:
-            model_type = model_config.hf_config.model_type
-            if "deepseek" in model_type:
-                raise NotImplementedError(
-                    "ACL Graph does not support deepseek. Please "
-                    "adopt additional_config={'enable_graph_mode': True} "
-                    "to serve deepseek models with NPU graph mode on vllm-ascend with V1 engine."
-                    " Or set `enforce_eager=True` to use eager mode.")
-            elif "qwen" not in model_type:
-                logger.warning(
-                    "ACL Graph is currently experimental. Please "
-                    "raise an issue on https://github.com/vllm-project/vllm-ascend/issues"
-                    " if you encourage any Error")
+        check_ascend_config(vllm_config, enforce_eager)
```

Comment:
move CompilationLevel check to `check_ascend_config` as well to make sure torchiar/aclgraph/eager works as expect.